advisories/hackage/spacecookie/HSEC-2025-0004.md

```diff
@@ -0,0 +1,35 @@
+```toml
+
+[advisory]
+id = "HSEC-2025-0004"
+cwe = []
```

Comment:
please set `cwe = [23]`.

 CWE-23: Relative Path Traversal  - https://cwe.mitre.org/data/definitions/23.html

Reply:
Does CAPEC-126 also make sense? https://capec.mitre.org/data/definitions/126.html

Reply:
Yes, good pick-up.  Please add it.

Reply:
Done.